specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/examples/glossary.json

```diff
@@ -48,6 +48,7 @@
     },
     "200": {
       "headers": {
+        "retry-after": "30",
```

Comment:
I'm not sure it matters, but these headers are not specified in the swagger https://github.com/Azure/azure-rest-api-specs/blob/fd39065dbcf846bb4e6336620e0a789268418891/specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json#L813
Same for the documents and storage endpoints

Reply:
added the header now in the swagger.